doc/cephfs/snap-schedule.rst

```diff
@@ -197,6 +197,15 @@ this happens, the next snapshot will be schedule as if the previous one was not
 delayed, i.e. one or more delayed snapshots will not cause drift in the overall
 schedule.
 
+If a volume is deleted while snapshot schedules are active on the volume, then
```

Comment:
volume delete can warn about snap schedules being active (if any) and that could hint the user to restart ceph-mgr (since this change does not 100% fix the issue anyway).

Reply:
done ... added warning message to `mgr/volumes` when volume is successfully deleted